src/Bicep.Core/Semantics/Namespaces/AzNamespaceSymbol.cs

```diff
@@ -56,6 +56,17 @@ private static ObjectType GetSubscriptionReturnValue(IEnumerable<FunctionArgumen
                 new TypeProperty("displayName", LanguageConstants.String),
             });
         }
+
+        private static ObjectType GetSingleProvidersReturnValue()
```

Comment:
>GetSingleProvidersReturnValue [](start = 34, length = 29)

nit - Should this be`GetSingleProvidersReturnType`?

Reply:
Yup, updated